lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -26,6 +26,7 @@ class SelfPacedDateOverrideTest(ModuleStoreTestCase):
     """
 
     def setUp(self):
+        self.reset_setting_cache_variables()
```

Comment:
This fixes a weird test bug we were seeing.  Either our test was setting these cache variables, or it was causing tests to be executed in a different order, causing some other test to set them.  Regardless, resetting before and after each test is the safest behavior.